src/book/kitchen-sink/02-formatting-paragraphs.mdx

```diff
@@ -25,7 +25,7 @@ _single underscores_
 
 **double asterisks**
 
-This paragraph has some `code` in it.
+This paragraph has some `code` in it. This sentence has a keyboard shortcut: press <kbd>Enter</kbd> to confirm. To deny, press <kbd>Ctrl</kbd>+<kbd>E</kbd>.
```

Comment:
This seems unrelated?

Reply:
There's no place in the kitchen sink where we test `<kbd>`, and it actually belongs here on the inline formatting page.